test/modules/standard/memory/countMemory/countMemory.prediff

```diff
@@ -14,12 +15,20 @@ if (-r $memfile) {
     }
   }
 } else {
-    $platform = `$ENV{CHPL_HOME}/util/chplenv/chpl_platform.py --target`; chomp($platform);
     if ($platform eq "darwin") {
         $memFreeInBytes = `sysctl -n hw.memsize`; chomp($memFreeInBytes);
     }
 }
 
+# cygwin32 running on a 64 bit windows can have more than 4GB reported by
+# /proc/meminfo, but only ~4GB is actually accessible
+if ($platform eq "cygwin32") {
+    $maxMemInBytes = 0xffffffff;
+    if ($memFreeInBytes > $maxMemInBytes) {
+        $memFreeInBytes = $maxMemInBytes;
+    }
```

Comment:
We haven't had problems in those contexts so far.  I would imagine that a virtual linux32 box won't have the same problem, and that this is a bug specific to cygwin32's implementation of cpuinfo